extension/src/cli/git/reader.ts

```diff
@@ -9,6 +9,7 @@ import { isDirectory } from '../../fileSystem'
 export const autoRegisteredCommands = {
   GIT_GET_BRANCHES: 'getBranches',
   GIT_GET_COMMIT_MESSAGES: 'getCommitMessages',
```

Comment:
Noticed that we only get commit message for the current branch. Are we planning to get commit message for other branches in the future?